bin/ch_core.c

```diff
@@ -45,9 +45,10 @@ struct bind BINDS_DEFAULT[] = {
    { "/sys",                     "/sys",                     BD_REQUIRED },
    { "/etc/hosts",               "/etc/hosts",               BD_OPTIONAL },
    { "/etc/machine-id",          "/etc/machine-id",          BD_OPTIONAL },
+   { "/etc/opt/cray",            "/etc/opt/cray",            BD_OPTIONAL },
    { "/etc/resolv.conf",         "/etc/resolv.conf",         BD_OPTIONAL },
-   { "/var/lib/hugetlbfs",       "/var/opt/cray/hugetlbfs",  BD_OPTIONAL },
```

Comment:
Double-checking we don't need `/var/lib/hugetlbfs` any more?

Reply:
The MPI cray-gni injection examples all pass on our XC systems without it. That said, if there is no harm in keeping them, perhaps this should be reverted.